meteor/utils.py

```diff
@@ -14,10 +14,10 @@ class ShapeMismatchError(Exception): ...
 
 
 @dataclass
-class MapLabels:
+class MapColumns:
     amplitude: str
     phase: str
-    uncertainty: str | None = None
+    uncertainty: str
```

Comment:
actually I think we should revert this back to `str | None`, since uncertainties are not required